CONTRIBUTION.md

```diff
@@ -4,78 +4,77 @@ When contributing to this repository, please first discuss the change you wish t
+* Once the actions pass on the main branch, delete the `develop-<feature/fix-name>` branch
 
-1. Install [package dependencies](https://sage-schematic.readthedocs.io/en/develop/README.html#installation-requirements-and-pre-requisites).
-2. Clone the `schematic` package repository.
+### Updating readthedocs documentation
+1. Navigate to the docs directory.
+2. Run make html to regenerate the build after changes.
+3. Contact the development team to publish the updates.
 
-```
-git clone https://github.com/Sage-Bionetworks/schematic.git
-```
+*Helpful resources*:
 
-3. [Create and activate](https://sage-schematic.readthedocs.io/en/develop/README.html#virtual-environment-setup) a virtual environment.
-4. Run the following commands to build schematic and install the package along with all of its dependencies:
+1. [Getting started with Sphinx](https://www.sphinx-doc.org/en/master/usage/quickstart.html)
+2. [Installing Sphinx](https://www.sphinx-doc.org/en/master/usage/installation.html)
 
-```
-cd schematic  # change directory to schematic
-git checkout develop  # switch to develop branch of schematic
-poetry build # build source and wheel archives
-pip install dist/schematicpy-x.y.z-py3-none-any.whl  # install wheel file
-```
-
-*Note*: Use the appropriate version number (based on the version of the codebase you are pulling) while installing the wheel file above.
-
-5. [Obtain](https://sage-schematic.readthedocs.io/en/develop/README.html#obtain-google-credentials-file-s) appropriate Google credentials file(s).
-6. [Obtain and Fill in](https://sage-schematic.readthedocs.io/en/develop/README.html#fill-in-configuration-file-s) the `config.yml` file and the `.synapseConfig` file as well as described in the `Fill in Configuration File(s)` part of the documentation.
-7. [Run](https://docs.pytest.org/en/stable/usage.html) the test suite.
+### Update toml file and lock file
+If you install external libraries by using `poetry add <name of library>`, please make sure that you include `pyproject.toml` and `poetry.lock` file in your commit.
 
-*Note*: To ensure that all tests run successfully, contact your DCC liason and request to be added to the `schematic-dev` [team](https://www.synapse.org/#!Team:3419888) on Synapse.
+### Code style
 
-8. To test new changes made to any of the modules within `schematic`, do the following:
+To ensure consistent code formatting across the project, we use the `pre-commit` hook. You can manually run `pre-commit` across the respository before making a pull request like so:
 
 ```
-# make changes to any files or modules
-pip uninstall schematicpy  # uninstall package
-poetry build
-pip install dist/schematicpy-x.y.z-py3-none-any.whl  # install wheel file
+pre-commit run --all-files
 ```
 
+Further, please consult the [Google Python style guide](http://google.github.io/styleguide/pyguide.html) prior to contributing code to this project.
+Be consistent and follow existing code conventions and spirit.
+
 ## Release process
```

Comment:
@jaymedina I see you didn't make any changes to this section so this comment isn't for you it's just to note that this section is out of date and will need to be updated in the future

Reply:
Sounds good! Did you want me to make a Jira ticket for this so we have it in our radar?

Reply:
I made a Jira ticket just in case! https://sagebionetworks.jira.com/browse/FDS-2516

Reply:
@jaymedina thank you! I thought I had done so but it looks like I hadn't. Thanks for creating!

Reply:
No worries! It looks like there's a ruleset in this repo that requires your approval before merge since there have been changes requested. I'm going to re-request a review. Thanks in advance!

Reply:
@jaymedina yep no problem!